admin/release-revert

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+## Revert local repos to pre-release state
+echo -e "${BOLD}${COLOR}CodeIgniter4 release revert${NORMAL}"
+echo '---------------------------'
+
+if [ $# -lt 2 ]; then
+    echo "You forgot the magic word"
+    exit 1
+fi
+if [ $1 != 'please' ]; then
```

Comment:
You have to say please to revert? What a polite script :)

Reply:
That is only to prevent inadvertent reverting. The other scripts need a version and/or qualifier.

Reply:
First time I read, I ask: "what magic word?" haha amazing